content/src/main/content/jcr_root/apps/acs-commons/components/utilities/qr-code-generator/clientlibs/js/script.js

```diff
@@ -0,0 +1,52 @@
+/*global JSON: false, angular: false */
+
```

Comment:
could this all be wrapped in an anonymous namespace? doubt well collide with another `qrCode` var, but its within the real of possibility

`$(function() { .... the contents of script.js });`